templates/fragments/osb.conf

```diff
@@ -11,6 +11,15 @@ location ~* /(s3fs-css|s3fs-js|${DRUPAL_PUBLIC_FILES_PATH})/(.*) {
     # Here we can include any custom location fragment.
     include /etc/nginx/conf.d/fragments/location/osb/*.conf;
     proxy_pass $bucket_base_url/$file_path$is_args$args;
+
+    #Hide Headers comin up from the storage bucket. 
+    proxy_hide_header "x-guploader-uploadid";
+    proxy_hide_header "x-goog-generation";
+    proxy_hide_header "x-goog-metageneration";
+    proxy_hide_header "x-goog-metageneration";
+    proxy_hide_header "x-goog-stored-content-length";
+    proxy_hide_header "x-goog-hash";
+    proxy_hide_header "x-goog-storage-class";
```

Comment:
generally speaking this kind of stuff should be toggled by a condition, there could be cases where we would like to have those variables (for example in a dev environment).

We should use an approach like this one:

https://github.com/sparkfabrik/docker-php-drupal-nginx/blob/feature/d8/templates/default.conf#L37
```
if [ ${NGINX_HTTPSREDIRECT} == 1 ]; then
  print "Enabling HTTPS redirect"
  sed  -e '/#httpsredirec/r /templates/httpsredirect.conf' -i /templates/default.conf;
  sed  -e '/#httpsredirec/r /templates/httpsredirect.conf' -i /templates/subfolder.conf;
fi
```

Reply:
Understood. I'm going with it. Thanks